sdks/java/testing/nexmark/src/main/java/org/apache/beam/sdk/nexmark/queries/Query10.java

```diff
@@ -314,7 +315,7 @@ public void processElement(ProcessContext c, BoundedWindow window)
             name + ".WindowLogFiles",
             Window.<KV<Void, OutputFile>>into(
                     FixedWindows.of(Duration.standardSeconds(configuration.windowSizeSec)))
-                .triggering(AfterWatermark.pastEndOfWindow())
+                .triggering(DefaultTrigger.of())
```

Comment:
```suggestion
```

Reply:
I will run spotlessApply.

Reply:
(did nothing - the formatting is correct)